notify/opsgenie/opsgenie.go

```diff
@@ -43,8 +43,8 @@ type Notifier struct {
 }
 
 // New returns a new OpsGenie notifier.
-func New(c *config.OpsGenieConfig, t *template.Template, l log.Logger) (*Notifier, error) {
-	client, err := commoncfg.NewClientFromConfig(*c.HTTPConfig, "opsgenie", false, false)
```

Comment:
The boolean flags in the previous version of `commoncfg.NewClientFromConfig()` were related to `disableKeepAlives, enableHTTP2 bool`, so `false, false` did mean "keepalive enabled, HTTP2 disabled". Keepalive and HTTP2 are now enabled by default so, to keep the same exact behaviour, I'm adding `WithHTTP2Disabled()`.

_Same comment applies to all other receiver integrations._